src/migrationdata/README.md

```diff
@@ -1,8 +0,0 @@
-## Purpose
```

Comment:
I liked this README!

Reply:
a little confused because I do not remember making this change and noticed some files changed that were not intended to be. checked out to their original form and the diff should include the actual ticket changes - thank you for pointing it out!